### PR TITLE
DOC-10786: Updates to REST connection management

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

**Comment by anuthan** [comment]
@Peter-Searby can you review the docs for Tony, thanks.

File: modules/rest-api/pages/rest-manage-cluster-connections.adoc [hunk 1]
```diff
@@ -68,9 +85,10 @@ If successful, the call returns an object such as the following:
 The following example sets the maximum number of connections for memcached to `650000`:
 
 ----
-curl -u Administrator:password -X POST \
-localhost:8091/pools/default/settings/memcached/global \
--d  max_connections=650000
+curl -X POST http://localhost:8091/pools/default/settings/memcached/global \
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-manage-cluster-connections.adoc:88]: Inconsistent use of the verbose flag: -v
It's probably best to use it either for all examples or none

File: modules/rest-api/pages/rest-manage-cluster-connections.adoc [hunk 2]
```diff
@@ -68,9 +85,10 @@ If successful, the call returns an object such as the following:
...
+-u Administrator:password \
+-d max_connections=650000 \
+-u Administrator:password
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-manage-cluster-connections.adoc]: duplicate -u Administrator:password

File: modules/rest-api/pages/rest-manage-cluster-connections.adoc [hunk 3]
```diff
@@ -94,6 +112,25 @@ If successful, the call returns an object such as the following:
...
+It also specifies the value of `connection_limit_mode` as `recycle`, to ensure that new connections are accepted; and specifies a `free_connection_pool_size` of `10000`.
+
+----
+curl -v -X POST http://localhost:8091/pools/default/settings/memcached/global \
+-d free_connection_pool_size=10000 \
+-u Administrator:password \
+-d connection_limit_mode=recycle \
+-d system_connections=750000 \
+-d max_connections=650000 \
+-d free_connection_pool_size=10000
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-manage-cluster-connections.adoc]: duplicate -d free_connection_pool_size=10000

File: modules/rest-api/pages/rest-manage-cluster-connections.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/rest-manage-cluster-connections.adoc:88]: Thanks, removed.

File: modules/rest-api/pages/rest-manage-cluster-connections.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/rest-manage-cluster-connections.adoc]: Thanks, removed.

File: modules/rest-api/pages/rest-manage-cluster-connections.adoc (hunk 3 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/rest-manage-cluster-connections.adoc]: Thanks, removed.

**Pull request merged**